c2corg_ui/static/partials/feed.html

```diff
@@ -1,76 +1,80 @@
-        </a>
-        <a class="thumbnail flex" ng-href="/images/{{::doc.image2.document_id}}/{{::doc.image2.locales[0].lang}}"
-         ng-if="::doc.image2">
-          <span class="image" style="background-image: url({{feedCtrl.createImageUrl(doc.image2.filename, 'MI')}})"></span>
-        </a>
-        <a class="thumbnail flex" ng-href="/images/{{::doc.image3.document_id}}/{{::doc.image3.locales[0].lang}}"
-          ng-if="::doc.image3">
-          <span class="image" style="background-image: url({{feedCtrl.createImageUrl(doc.image3.filename, 'MI')}})"></span>
-        </a>
+    <div class="in-feed-col-2">
+      <article ng-repeat="doc in feedCtrl.documentsCol[1] track by doc.id">
+        <app-card app-card-doc="::doc"></app-card>
+      </article>
+    </div>
+    <div class="in-feed-col-3">
+      <article ng-repeat="doc in feedCtrl.documentsCol[2] track by doc.id">
+        <app-card app-card-doc="::doc"></app-card>
+      </article>
+    </div>
+    <div style="clear:both">&nbsp;</div>
+    <h4 class="text-center text-danger" translate ng-show="feedCtrl.error">Sorry, there was an error while getting the feed.</h4>
+    <h4 class="text-success text-center" translate ng-show="feedCtrl.busy">Loading feed, please wait...</h4>
+    <h4 class="text-success text-center" translate ng-show="feedCtrl.feedEnd">No more feed to load.</h4>
+  </div>
+</div>
+<div class="feed-cr">
+  <div class="info" ng-show="feedCtrl.showAssoBlock">
+    <h3><span translate>Bienvenue sur Camptocamp.org!</span> <span class="glyphicon glyphicon-remove pull-right" ng-click="feedCtrl.toggleBlock(0)"></span></h3>
+    <div class="finfo">
+      <div class="ficontent">
+        <div class="fleft">
+          <div class="ftxt">
+            <div class="ftitle">
+              <p class="title">Camptocamp.org est un site communautaire dédié aux sports de montagne. Le site est géré par une association à but non lucratif, qui a fait le choix de contenus sous licences libres. N'hésitez pas à vous inscrire et à contribuer ! <br><span class="center"><a href="/articles/106726/fr/camptocamp-association" title="Rejoignez l'association">Envie d'en savoir plus ?</a> - <a href="https://forum.camptocamp.org/c/site-et-association" title="Association Camptocamp">Forum de l'association</a></span></p>
+            </div>
+          </div>
+        </div>
       </div>
-      <div class="list-item {{::type}}s" ng-init="feed = true">
-        <app-card app-card-doc="::doc['document']"></app-card>
+    </div>
+  </div>
+  <div class="info" ng-show="feedCtrl.showMobileBlock">
+    <h3><span translate>Application mobile</span> <span class="glyphicon glyphicon-remove pull-right" ng-click="feedCtrl.toggleBlock(1)"></span></h3>
+    <div class="finfo">
+      <div class="ficontent noflex">
+        <p>Retrouver tout Camptocamp en montagne avec la nouvelle application Smartphone.</p>
```

Comment:
Please use an english text and add a ``translate`` attribute to the ``<p>`` element.